ZUGFeRD/Party.cs

```diff
@@ -23,18 +23,19 @@
 
 namespace s2industries.ZUGFeRD
 {
-    public class Party
-    {
-        public string ID { get; set; }
-        public string Name { get; set; }
-        public string City { get; set; }
-        public string Postcode { get; set; }
-        public CountryCodes Country { get; set; }
+   public class Party
+   {
+      public string ID { get; set; }
+      public string Name { get; set; }
+      public string Receiver { get; set; }

```

Comment:
In LineOne wenn Receiver gefüllt ist. Dann wandert Street in LineTwo.
Sonst enfällt es und Street bleibt in LineOne.